sql/0_node.sql

```diff
@@ -48,28 +49,12 @@ select 'Started At', pg_postmaster_start_time()::timestamptz(0)::text
 union all
 select 'Uptime', (now() - pg_postmaster_start_time())::interval(0)::text
 union all
-select
-  'Checkpoints',
```

Comment:
removing some reports is not good - we need to support them in different versions (like we do it for pgss reports)